app/src/main/java/ch/epfl/sdp/map/MapsActivity.java

```diff
@@ -120,58 +122,67 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         locationFinder = new GoogleLocationFinder((LocationManager) getSystemService(Context.LOCATION_SERVICE));
     }
 
+    /**
+     * on Travis the map will not appear since it is not connected via API Key
+     * @param googleMap
+     */
     @Override
     public void onMapReady(GoogleMap googleMap) {
-        Game.getInstance().setMapApi(new GoogleMapApi(googleMap));
-        Game.getInstance().setRenderer(this);
-
-        //Get email of CurrentUser;
-        String email = authenticationAPI.getCurrentUserEmail();
-
-        Log.d("Database", "Game running is " + Game.getInstance().isRunning());
-
-        if (!Game.getInstance().isRunning()) {
-            commonDatabaseAPI.fetchUser(email, fetchUserRes -> {
-                if (fetchUserRes.isSuccessful()) {
-                    Player currentUser = EntityConverter.userForFirebaseToPlayer(fetchUserRes.getResult());
-                    playerManager.setCurrentUser(currentUser);
-                    Game.getInstance().addToDisplayList(currentUser);
-                    Log.d("Database", "User fetched");
+        if (((MyApplication)getApplication()).appContainer.testing) { return; }

```

Comment:
I think you can remove the testing here, as I think this method is not executed in Travis. And if it is, then we only need to set some dependencies in `@Before` (the mapApi and the renderer).